src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -100,7 +100,9 @@ private void OnFileSaved(object sender, TextDocumentFileActionEventArgs e)
 
             if (e.FileActionType == FileActionTypes.ContentSavedToDisk && textDocument != null)
             {
-                _ = Task.Run(async () =>
+                _ = Task.Run(async () => await DoRestoreOnSaveAsync());
```

Comment:
>_ = Task.Run(async () => await DoRestoreOnSaveAsync()); [](http://example.com/codeflow?start=16&length=55)

It's late, but what's the functional difference here?

Reply:
Functionally no difference, I just converted it to a local function while I was debugging and overall prefer a local function over lengthy anonymous delegates.  It does make the stack trace more identifiable.  I could revert this but felt it was a slight overall improvement to the code.

Reply:
I have no problem with it, it was late, I was a *couple* beers in and just couldn't see why it was done.